features/cisSshd/exec.config

```diff
@@ -1,5 +1,102 @@
 #!/usr/bin/env bash
 set -Eeuo pipefail
 
-# 5.2.1: Define sshd conf permission
+# Define sshd conf permission
 chmod 600 /etc/ssh/sshd_config
+
+# Append sshd_config
+cat << 'EOF' >> /etc/ssh/sshd_config
+
+#### CIS-HARDENING + CUSTOM ####
```

Comment:
what does 'custom' mean?

it looks like a lot of this config is duplicated from what we have anyway in any image with the ssh feature. do we really need this?